backend/config.go

```diff
@@ -151,6 +156,74 @@ func (c *GrafanaCfg) ConcurrentQueryCount() (int, error) {
 	return i, nil
 }
 
+type SqlConfig struct {
+	RowLimit                      int64
+	DefaultMaxOpenConns           int
+	DefaultMaxIdleConns           int
+	DefaultMaxConnLifetimeSeconds int
+}
+
+func (c *GrafanaCfg) Sql() (SqlConfig, error) {
```

Comment:
what do we normally use `Sql` or `SQL`?  :D

Reply:
yes 😁 

no idea, i quickly checked, and found these:
```
	SqlDatasourceMaxOpenConnsDefault    int
	SqlDatasourceMaxIdleConnsDefault    int
	SqlDatasourceMaxConnLifetimeDefault int
```
https://github.com/grafana/grafana/blob/6ce0efeb41e7c9c4392f439a9b10e23486e2a094/pkg/setting/setting.go#L345-L347

Reply:
seems like the CI wants it to be `SQL` :) https://drone.grafana.net/grafana/grafana-plugin-sdk-go/1847/1/3

Reply:
argh.. ok, i'll rename those.. but what about the fields of the structure.. should i switch those to `SQL` too?

Reply:
I think so, yes, if not you are going to get lint errors.

Reply:
thanks, done!